servers/src/test/java/tachyon/master/file/meta/AbstractInodeTest.java

```diff
@@ -20,12 +20,16 @@
 
 import tachyon.Constants;
 import tachyon.master.block.BlockId;
+import tachyon.master.file.meta.InodeDirectory;
+import tachyon.master.file.meta.InodeFile;
+import tachyon.security.authorization.PermissionStatus;
 
 /**
  * Abstract class for serving inode tests.
  */
 public abstract class AbstractInodeTest {
-
+  private static PermissionStatus sPermissionStatus =
+      new PermissionStatus("user1", "group1", (short)0755);
```

Comment:
define these values as public constants that the other tests access to avoid duplicating constants:

```
public static String TEST_USER_NAME = "user1"
...
```

so that, for instance InodeDirectoryTest can do:

```
Assert.assertEquals(AbstractInodeTest.TEST_USER_NAME, inode2.getUsername())
```